docs/scarpet/api/Auxiliary.md

```diff
@@ -142,6 +142,33 @@ Available shapes:
      * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
      * `level` - level of details, see `'sphere'`.
 
+ * `'poly'`:
+   * Required attributes:
+     * `points` - list of points defining vertices of the polygon
+   * Optional attributes:
+     * `relative` - list of bools. vertices of the polygon that affected by 'follow'. Default means that every point is affacted.
```

Comment:
I would guess, could be just a single bool to affect all points, right?

Reply:
no. but default means that every point is affacted. if 'follow' is omited, then every point is not affacted. so i dont see why it need to support a single bool.

but if you really want, i could add it.

Reply:
> I would guess, could be just a single bool to affect all points, right?

ok. i let it could be a single bool to affect all points now.